enrollment_predictions/models/course_model.py

```diff
@@ -225,7 +248,7 @@ def main():
     from sklearn.metrics import mean_squared_error
```

Comment:
Please move imports to the top for formatting

Reply:
Will do this in the next PR